src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,51 @@
 import React from 'react';
-
+import { Movie } from '../../types/Movie';
 import './MovieCard.scss';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
+export const MovieCard: React.FC<Movie> = ({
```

Comment:
you can export `FC` from `react` directly
import React, { FC } from 'react';
```suggestion
export const MovieCard: FC<Movie> = ({
```